content/en/about/_index.html

```diff
@@ -1,30 +1,90 @@
 ---
-title: About
-linkTitle: About
+title: Build
+linkTitle: Build
 menu:
   main:
     weight: 10
 
 ---
-
 <section class="of-masthead of-masthead--subpage">
-  <h1 class="of-heading of-heading--xl">About OLM</h1>
+  <h1 class="of-heading of-heading--xl">Building Operators using the Operator SDK</h1>
```

Comment:
is this intentional? converting the copy over to sdk?

Reply:
@ecordell yes. The about section wasn't doing much for us, it just seemed like a complete waste of space we can utilize instead to draw attention to the other of project.